tests/test_problems/test_zoo/test_diffeq/test_ivp_examples_jax.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
```

Comment:
I know this test script may be a bit confusing. Please let me know if it is too bad. Maybe someone with more pytest knowledge will be able to improve it?!

Reply:
You should run the tests that do not use JAX even if jax is available in my opinion since the CI toolchain will import jax and then these tests will never be run I think.

Reply:
The tests below check that *if jax is not available, the import error is thrown*. On the other hand, the other tests check that *if jax is available, code is being executed*. So they check the same functions in two separate scenarios, and I dont know how to run them anyway here.

Reply:
Though if you have any idea how to handle this better, I'd gladly make changes :)

Reply:
Let me try to explain. Currently the CI imports jax (except on windows) and runs the tests that are not skipped if `JAX_AVAILABLE`. However, in my understanding this  means that tests for code that is run if jax is not available (in this case throwing an import error) is actually not tested by the CI. Judging from the coverage this seems to be the case. Here this is not much of a problem but should you have more sophisticated code that users without jax might want to run, then this should be tested explicitly in my opinion. There is probably two ways to do this, either have different test configurations (with and without jax available) or simply tests that call functions which are called when jax is not available explicitly, sth like `diffeq_zoo.threebody_nojax()`. Since in this PR this simply results in an error I think this is good to go, but we might want to keep this in mind in the future.

Reply:
Ah okay. Yes, I agree with everything said. Let's keep this in mind for the future!